kimchi/src/tests/chacha.rs

```diff
@@ -71,9 +71,13 @@ fn chacha_prover() {
 
     // create the index
     let fp_sponge_params = oracle::pasta::fp::params();
-    let cs =
-        ConstraintSystem::<Fp>::create(gates, vec![chacha::xor_table()], fp_sponge_params, PUBLIC)
-            .unwrap();
+    let cs = ConstraintSystem::<Fp>::create(
+        gates,
+        vec![chacha::testing::xor_table()],
```

Comment:
The `xor_table` is an inherent part of the `chacha` gate, not just for testing. It shouldn't have been moved to a testing module (and this PR isn't the place to do that anyway).